content/livesync/how-livesync-works.textile

```diff
@@ -0,0 +1,95 @@
+---
+title: How LiveSync works
+meta_description: "LiveSync enables you to synchronize changes in your database to frontend clients at scale."
+product: livesync
+---
+
+LiveSync is a powerful realtime data synchronization feature designed to facilitate the synchronising of changes from backend databases to frontend clients. Leveraging Pub/Sub Channels, LiveSync ensures that data updates are propagated reliably to all connected clients in realtime.
+
+LiveSync benefits applications requiring realtime collaboration, such as collaborative form editing, online Auction, task management applications, or simple multiplayer turn-based games, where maintaining data consistency and low latency are paramount. Developers can build highly responsive and interactive applications that deliver an exceptional realtime user experience.
```

Comment:
I think this reads like marketing speak. Additionally, I think we are missing some context in terms of the context flow before jumping into channel based broadcasting.

Reply:
I agree there is bit of jump from this paragraph to the next part. Perhaps I high level overview of the the pattern. A write to an outbox table, this is picked up by our ADBC and published to the ably and then onto the subscribers.

Reply:
I've updated this by adding another paragraph following this that gives an overview of how LiveSync works each step of the way.